new/src/constants/events.tsx

```diff
@@ -0,0 +1,34 @@
+import MainEventMainIcon from "../assets/event-icons/laptop.svg";
+import MainEventSideIcon from "../assets/event-icons/target.svg";
+import EventIcon1 from "../assets/event-icons/tabs-1.svg";
+import EventIcon2 from "../assets/event-icons/tabs-2.svg";
+
+export const MAIN_EVENT = {
+  name: "GameCraft 2023",
+  details: ["Dec 2023", "Game Jam", "7 Days", "50+ Participants"],
+  link: "https://gamecraft.nusgdg.org/",
+  mainIcon: MainEventMainIcon,
+  sideIcon: MainEventSideIcon,
+  date: new Date("2023-12-09"),
```

Comment:
1. Please change this to `Gamecraft! 2024`, with an exclamation mark in the middle and year 2024.
2. Please update the details accordingly. Tentative date for this year gamecraft is 9 Dec 2024.